vaadin-combo-box-flow-integration-tests/src/test/java/com/vaadin/flow/component/combobox/test/LazyLoadingIT.java

```diff
@@ -414,7 +425,10 @@ public void setComponentRenderer_scrollDown_scrollUp_itemsRendered() {
         clickButton("component-renderer");
         beanBox.openPopup();
         scrollToItem(beanBox, 300);
+        
+        waitUntil(e -> getOverlayContents().contains("<flow-component-renderer appid=\"ROOT\"><h4>Person 300</h4></flow-component-renderer>"));
```

Comment:
It would be best to avoid relying on `<flow-component-renderer>` since it's an implementation detail, e.g:
```java
getOverlayContents().stream().anyMatch(s -> s.contains("<h4>Person 300</h4>"))
```

Reply:
Done